pypit/arsort.py

```diff
@@ -465,7 +475,10 @@ def match_science(fitsdict, filesort):
                 # Select the closest calibration frames to the science frame
                 tdiff = np.abs(fitsdict['time'][n].astype(np.float64)-np.float64(fitsdict['time'][iSCI[i]]))
                 wa = np.argsort(tdiff)
-                settings.spect[ftag[ft]]['index'].append(n[wa[:numfr]])
+                if settings.argflag['run']['calcheck']:
+                    settings.spect[ftag[ft]]['index'].append(n[wa])
+                else:
+                    settings.spect[ftag[ft]]['index'].append(n[wa[:numfr]])
```

Comment:
Shouldn't we always use this line? That way the output files will list exactly the numfr files to be used for the reduction.